packages/node-cli/src/SdkExecutor.js

```diff
@@ -59,6 +59,16 @@ module.exports = class SdkExecutor {
 		});
 	}
 
+	// Taken from https://medium.com/@almenon214/killing-processes-with-node-772ffdd19aad
+	_killChildProcessAndItsChildren() {
+		NodeConsoleLogger.info(`About to kill childProcess.pid=${this.childProcess.pid} and all of its sub processes`);
+		if (isWin) {
+			exec(`taskkill /PID ${this.childProcess.pid} /T /F`);
+		} else {
+			process.kill(-this.childProcess.pid);
```

Comment:
Discuessed f2f the risk of doing this. We aren't waiting to verify that the process does in fact close.
The proper way of doing this is sending a sigTerm waiting and then sending a sigKill.
This might be slightly complicated because what will happen if the parent dies and only a few children remain, will this syntax work?
I think it is reasonable to add a comment, open a ticket and merge this way. We will have the same symptom we are having now (orphaned child processes).